assets/src/scss/layout/_navbar.scss

```diff
@@ -261,34 +341,32 @@ a.nav-link {
     transform: scaleX(0);
     transition: transform 0.25s;
     z-index: -1;
+
+    @include large-and-up {
+      border-bottom-width: 4px;
+    }
   }
 }
 
 .active a.nav-link {
   --nav-link-active-- {
-    color: var(--nav-link--color, $white);
-    text-decoration-color: $gp-green;
-  }
+    color: $white;
 
-  opacity: 1;
-  text-decoration-line: underline;
-  text-decoration-thickness: 3px;
-  text-underline-offset: auto;
-  text-underline-position: under;
+    &:hover {
+      color: $white;
+    }
+  }
 
-  @include large-and-up {
-    text-decoration-line: none;
-    text-decoration-thickness: 4px;
+  &:before {
```

Comment:
I think you need to add `transform: scaleX(1);` here too otherwise the border doesn't show for the active item